settings.gradle

```
@@ -11,12 +11,42 @@ plugins {
     id 'com.gradle.common-custom-user-data-gradle-plugin' version '1.11.1'
 }
 
+import com.gradle.scan.plugin.PublishedBuildScan
+
 gradleEnterprise {
     server = "https://ge.armeria.dev"
     buildScan {
-        publishIfAuthenticated()
-        publishAlways()
-        uploadInBackground = System.getenv("CI") == null
+        if (System.getenv("CI") == null) {
+            publishIfAuthenticated()
+            publishAlways()
+        } else {
+            uploadInBackground = false
```

We'd still suggest to always publish build scans.

```suggestion
        def isCi = System.getenv("CI") != null
        publishIfAuthenticated()
        publishAlways()
        uploadInBackground = !isCi
        if (isCI) {
```

---

I intended to disable build scans for tasks run in CI builds so that we can publish later with `buildScanPublishPrevious`.
Are the build scans automatically deactivated if `GRADLE_ENTERPRISE_ACCESS_KEY` is null?